### PR TITLE
Added entry for difference in sleep values

### PR DESCRIPTION
**Pull request by gwright99**

### CODE REVIEW

File: docs/faqs.md [hunk 1]
```diff
@@ -118,6 +118,15 @@ Please verify the following:
 This error can occur if you execute a DSL 1-based Nextflow workflow using [Nextflow 22.03.0-edge](https://github.com/nextflow-io/nextflow/releases/tag/v22.03.0-edge) or later.
 
 
+**<p data-question>Q: Does the sleep command work the same way across my entire script?</p>**
+
+The `sleep` command behaves slightly differently depending on where it is executed in your Nextflow script:
+
+- If used within an `errorStrategy` block, the Groovy sleep function will be used (which takes its value in milliseconds).
+- If used within a non-Groovy process script block ([example](https://www.nextflow.io/docs/latest/metrics.html?highlight=sleep)), the BASH sleep function will be used (which takes its value in seconds).
+
```
Comment (pditommaso) [inline docs/faqs.md:127]: This seems to suggest it's the same `sleep` behaving behaving differently. It should be pointed out the `sleep` in the task has nothing to do with the nextflow/script `sleep`.

Comment (gwright99) [reply docs/faqs.md]: Text updated to make the distinction clearer.

**Comment by abhi18av** [comment]
CC @llewellyn-sl

File: docs/faqs.md [hunk 2]
```diff
@@ -118,6 +118,14 @@ Please verify the following:
 This error can occur if you execute a DSL 1-based Nextflow workflow using [Nextflow 22.03.0-edge](https://github.com/nextflow-io/nextflow/releases/tag/v22.03.0-edge) or later.
 
 
+**<p data-question>Q: Does the sleep command work the same way across my entire script?</p>**
+
+The `sleep` commands within your Nextflow workflows may differ in behaviour depending on where they are:
+
+- If used within an `errorStrategy` block, the Groovy sleep function will be used (which takes its value in milliseconds).
+- If used within a process script block, that language's sleep binary/method will be used. **Example:** [this BASH script](https://www.nextflow.io/docs/latest/metrics.html?highlight=sleep) uses the BASH sleep binary,  which takes its value in seconds.
```
Comment (pditommaso) [inline docs/faqs.md:126]: It's `Bash` not `BASH` (sorry, I've learnt this from a Nextflow docs contributor 😄)

Comment (gwright99) [reply docs/faqs.md:126]: And now I know too! Will change the capitalization.

**Pull request merged**